### ISSUE TITLE
Retrieve emails by subject - RestAPI

### ISSUE THREAD

**Issue by 1dson** [maildev/maildev#330]
From the README it's not that clear but is it possible to retrieve an email by subject using the REST API?

**Comment by djfarrelly** [comment]
Yes — you can filter by subject via the REST API using a query param on the list endpoint:

```
GET /api/email?subject=<exact subject>
```

Multiple params are ANDed, and you can also filter by other fields (e.g. `to.address`, `from.address`). See the `/api/email` handler in `packages/api/src/server.ts`.

One caveat: matching is **exact string equality**, not substring/partial. For fuzzy/substring subject search, the MCP layer's `maildev_search_emails` tool does that. Closing as answered — if you'd like substring subject matching in the REST endpoint specifically, please open a fresh issue for it.
